ui/src/app/apps/apps.component.html

```diff
@@ -8,15 +8,18 @@ <h1 [ngBusy]="busy">Apps</h1>
 ><span class="glyphicon glyphicon-plus"></span>
     Register Application(s)
 </button>
-<scdf-tri-state-button *ngIf="appRegistrations?.items" class="toggle-all" [items]="appRegistrations.getItemsAsObservable()"></scdf-tri-state-button>
+<scdf-tri-state-button *ngIf="appRegistrations?.items" class="toggle-all" [items]="appRegistrations.getItemsAsObservable()" (click)="unregisterMultipleApps(appRegistrations.items)"></scdf-tri-state-button>
```

Comment:
When button is disabled because 0 items are selected for deletion.     It still acts on button click.  Do we need to handle this in the unregisterMultipleApps()?